lib/iris/plugins/README.md

```diff
@@ -0,0 +1,4 @@
+# Iris plugins
```

Comment:
Could this also include a link to the Iris documentation page/section on plugins?

Reply:
I think it's appropriate to link to `latest`, since this README is for developers and we could be actively developing this section.

Appreciate that the link will be broken until merged, but I'm comfortable with that.